portal/server/helpers/getUserRoles.test.js

```diff
@@ -11,14 +11,28 @@ describe('getUserRoles', () => {
       });
 
       it(`should return false when roles does NOT include ${role}`, () => {
-        const result = getUserRoles(['']);
+        const result = getUserRoles([]);
```

Comment:
I'm not convinced about the way this test works. I think it could probably be replaced with a single test that just makes sure all the results are false if an empty array is passed in. Further, this test sounds more like you should pass in an array which includes all roles **expect** the role currently being tested. I'm curious to know what the rest of you think